inventory-generation/identity-management/main.yml

```diff
@@ -13,6 +13,10 @@
       include_vars:
         file: "{{ directory }}/engagement.json"
 
+    - name: Read Runtime Data
```

Comment:
Since "older" repos don't necessarily have the `runtime.json` file, we may want to add a check above and error out if it doesn't exist - see example above

Reply:
I've added check for the `runtime.json` file and failing the playbook if it's not exist. Probably failing is not a good option but I thing most of the engagements already since change for `runtime.json` file was deployed to production couple weeks ago.

The problem is what should be used for `user_options` and `user_role` variables as default values. Right now `residency` and `do500` having different values for `user_role` and these values important for the admin role.

Reply:
Added checks for user_options.user_roles.options (7fc8de944913224f7da12e8354ab8c7fd4ac2d82)